<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>746</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Gestion des Absences</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:40px</string>
    </property>
    <property name="text">
     <string>Gestion des Absences</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left:10px</string>
    </property>
    <property name="placeholderText">
     <string>Rechercher un étudiant...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>251</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Gestion</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>232</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ajouter/retirer absences :</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="minusTwo">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="minusOne">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addOne">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>+1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addTwo">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>+2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="edit">
        <property name="styleSheet">
         <string notr="true">font-size:17px</string>
        </property>
        <property name="text">
         <string>Modifier...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pdf">
        <property name="styleSheet">
         <string notr="true">font-size:17px</string>
        </property>
        <property name="text">
         <string>Générer PDF...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ajouterEt">
        <property name="styleSheet">
         <string notr="true">font-size:17px</string>
        </property>
        <property name="text">
         <string>Ajouter des étudiants...</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="supprimer">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:17px</string>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="toutSupp">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:17px</string>
          </property>
          <property name="text">
           <string>Tout effacer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>1001</width>
      <height>551</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView {
    selection-font-weight:bold;
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Niveau</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Filière</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Groupe</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Année étude</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Numéro stagiaire</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CIN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prénom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Absence</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Filtres Avancés</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="1">
       <widget class="QComboBox" name="cb">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-left:10px;</string>
        </property>
        <item>
         <property name="text">
          <string>Tout</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M101</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MA101</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MA102</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MA103</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MA104</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MA105</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PS101</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PS102</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PV101</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PV102</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EB101</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EB102</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EB103</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EB104</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EB106</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ATV201</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ATV202</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ATV203</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CCTP101</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CCTP201</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TDB101</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TDB102</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TDB201</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TDB202</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TDB203</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AA101</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AA102</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AA103</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AA104</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGE201</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGE202</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGE203</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGE204</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSCTTP101</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSCTTP102</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSCTTP201</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSCTTP202</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSCTTP203</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGO101</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGO102</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGO103</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGO104</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGO201</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGO202</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGT101</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGT102</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGT201</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGT202</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TSGT203</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GE101</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GE102</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GE103</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GE104</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GE105</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GE106</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GE107</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="limit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Activer le filtre absences</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>231</width>
       <height>67</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Max Absences :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Min Absences :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="min">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="max">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>56</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Groupe :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="previous">
       <property name="text">
        <string>Précédent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next">
       <property name="text">
        <string>Suivant</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>660</y>
      <width>193</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">color:#aaa</string>
       </property>
       <property name="text">
        <string>Développé par :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="me">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;
color:#0645AD;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Khamri Achraf</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>185</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Nombre de résultats :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="count">
       <property name="text">
        <string>Rien</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Importer depuis un fichier .csv</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
